app/services/PenaltiesService.scala

```diff
@@ -67,4 +83,10 @@ class PenaltiesService @Inject()(connector: PenaltiesConnector) {
   def findTotalLSPFromPayload(payload: ETMPPayload): BigDecimal = {
     payload.penaltyPoints.map(_.financial.map(_.amountDue)).collect{ case Some(x) => x }.foldRight(BigDecimal(0))(_ + _)
   }
+
+  def estimatedVATInterest(payload: ETMPPayload): (BigDecimal, Boolean) = {
+        val estimatedVAT = findEstimatedVatInterestFromPayload(payload)
+        val crystallisedVAT = findCrystalizedInterestFromPayload(payload)
+        Option(crystallisedVAT + estimatedVAT, estimatedVAT > 0).getOrElse(0,false)
```

Comment:
You don't need to wrap this in an Option. `estimatedVAT` and `crystallisedVAT` are not Option's so you can just have:
```suggestion
       (crystallisedVAT + estimatedVAT, estimatedVAT > 0)
```